examples/findfiles_up.cf

```diff
@@ -0,0 +1,39 @@
+#+begin_src prep
+#@ ```
+#@ mkdir -p /tmp/repo/.git/
+#@ touch /tmp/repo/.git/config
+#@ mkdir -p /tmp/repo/submodule/.git/
+#@ touch /tmp/repo/submodule/.git/config
+#@ mkdir -p /tmp/repo/submodule/some/place/deep/within/my/repo/
+#@ ```
+#+end_src
+###############################################################################
+#+begin_src cfengine3
+body common control
+{
+      bundlesequence  => { "example" };
+}
+
+bundle agent example
+{
+  vars:
+      "path" # path to search up from
+        string => "/tmp/repo/submodule/some/place/deep/within/my/repo/";
+      "glob" # glob pattern matching filename
+        string => ".git/config";
+      "level" # how far to search
+        int => "inf";
+      "configs"
+        data => findfiles_up("$(path)", "$(glob)", "$(level)");
+  reports:
+      "Submodules '$(glob)' is located in '$(configs[0])'";
+      "Parents '$(glob)' is located in '$(configs[1])'";
+}
+#+end_src
+###############################################################################
+#+begin_src example_output
+#@ ```
+#@ R: "Submodules '.git/config' is located in '/tmp/repo/.git/config'"
```

Comment:
You have double quotes around your strings here in the output, but those are not emitted.

i think it should be

```
#@ R: Submodules '.git/config' is located in '/tmp/repo/.git/config'
#@ R: Parents '.git/config' is located in '/tmp/repo/submodule/.git/config'
```